cli/maintainer/src/commands_anker.rs

```diff
@@ -246,6 +246,9 @@ struct ShowAnkerOutput {
     #[serde(serialize_with = "serialize_b58")]
     b_sol_mint_authority: Pubkey,
 
+    #[serde(serialize_with = "serialize_b58")]
+    token_swap_pool: Pubkey,
```

Comment:
If we put it in the struct, we should also update the `fmt::Display` impl below to print it.